src/components/Collapsible/__snapshots__/Collapsible.spec.jsx.snap

```diff
@@ -7,7 +7,9 @@ exports[`Collapsible [common] example testing should match snapshot(s) for 1.tog
       "height": 0,
     }
   }
-/>
+>
```

Comment:
enzyme shallow no longer uses a self closing tags, and now includes the immediate child's text/node